package/src/models/views/password.ts

```diff
@@ -68,6 +68,13 @@ const passwordConfig: AuthMachineConfig = {
           target: "showForm",
         },
         onDone: [
+          // If the user did not have a password previously, let them
+          // know that an email was sent to set their password.
+          {
+            cond: "passwordResetEmailSent",
+            actions: "setEmailSentMessage",
```

Comment:
The `passwordResetEmailSent` and `setEmailSentMessage` functions aren't defined yet.

The actions/guards that we refer to by name get hooked up to their corresponding functions in `src/models/forms/universal.ts` - they're the big lists of functions imported and used in `defaultOptions` at the top of the file. So for this to work I'd expect to see a `passwordResetEmailSent` entry added to `defaultOptions.guards`, and a `setEmailSentMessage` entry added to `defaultOptions.actions`. By convention they'd be defined with the same name in `src/models/config/guards.ts` and `actions.ts` respectively.

They can be defined inline, which I find useful when developing - it's good practice to extract them when they're finalized though.